secure_message/repository/database.py

```diff
@@ -39,7 +39,7 @@ class SecureMessage(db.Model):
     def __init__(self, msg_id="", subject="", body="", thread_id="", collection_case='',
                  ru_id='', survey='', collection_exercise='', from_internal=False):
 
-        logger.debug("Initialised Secure Message entity: msg_id: {}".format(id))
+        logger.debug(f"Initialised Secure Message entity: msg_id: {id}")
```

Comment:
[Out of scope of this PR] - I've raised an issue (https://github.com/ONSdigital/ras-secure-message/issues/155) to address that logs should be structured

Reply:
Awesome, I agree that's definitely a better way of doing it.